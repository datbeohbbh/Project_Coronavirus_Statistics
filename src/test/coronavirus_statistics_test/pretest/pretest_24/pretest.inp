26 27
q 1 17 455
q 1 26 65
? 4 2
q 1 18 374
? 4 1
q 1 14 296
q 1 8 213
? 4 5
? 6
? 4 2
? 4 1
q 3 18 320
q 3 8 64
q 2 17 242
q 3 8 8
? 4 3
? 7
? 4 5
? 6
? 6
q 1 25 299
? 4 1
? 7
? 4 3
? 6
? 4 3
? 6