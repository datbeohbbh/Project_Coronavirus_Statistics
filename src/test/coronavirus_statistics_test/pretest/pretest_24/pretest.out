Query ? 4 2 :
The answer is the city with the index 17 
- Information of the city 17 -
-> Number of current patients : 455
-> Has recovered : 0
-> Was dead : 0

Query ? 4 1 :
The answer is the city with the index 26 
- Information of the city 26 -
-> Number of current patients : 65
-> Has recovered : 0
-> Was dead : 0

Query ? 4 5 :
The answer is the city with the index 17 
- Information of the city 17 -
-> Number of current patients : 455
-> Has recovered : 0
-> Was dead : 0

Query ? 6 :
Currently, no recovery cases have been recorded

Query ? 4 2 :
The answer is the city with the index 8 
- Information of the city 8 -
-> Number of current patients : 213
-> Has recovered : 0
-> Was dead : 0

Query ? 4 1 :
The answer is the city with the index 26 
- Information of the city 26 -
-> Number of current patients : 65
-> Has recovered : 0
-> Was dead : 0

Query ? 4 3 :
The answer is the city with the index 8 
- Information of the city 8 -
-> Number of current patients : 141
-> Has recovered : 0
-> Was dead : 72

Query ? 7 :
The city with the highest death is city with the index 18
Quantity: 320 people

Query ? 4 5 :
The answer is the city with the index 14 
- Information of the city 14 -
-> Number of current patients : 296
-> Has recovered : 0
-> Was dead : 0

Query ? 6 :
The city with the highest recovery is city with the index 17
Quantity: 242 people

Query ? 6 :
The city with the highest recovery is city with the index 17
Quantity: 242 people

Query ? 4 1 :
The answer is the city with the index 18 
- Information of the city 18 -
-> Number of current patients : 54
-> Has recovered : 0
-> Was dead : 320

Query ? 7 :
The city with the highest death is city with the index 18
Quantity: 320 people

Query ? 4 3 :
The answer is the city with the index 8 
- Information of the city 8 -
-> Number of current patients : 141
-> Has recovered : 0
-> Was dead : 72

Query ? 6 :
The city with the highest recovery is city with the index 17
Quantity: 242 people

Query ? 4 3 :
The answer is the city with the index 8 
- Information of the city 8 -
-> Number of current patients : 141
-> Has recovered : 0
-> Was dead : 72

Query ? 6 :
The city with the highest recovery is city with the index 17
Quantity: 242 people

